.travis.yml

```diff
@@ -7,7 +7,7 @@ rvm:
 - 2.4
 - 2.5
 - 2.6
-- ruby-head
```

Comment:
should still keep ruby-head but allow it to fail

Reply:
You can add the following to the config to allow `ruby-head` to fail.
```
matrix:
  allow_failures:
  - rvm: ruby-head
```

Reply:
Oh cool! Good to know!